sample-libraries/a/build.gradle

```diff
@@ -18,14 +18,14 @@ apply plugin: 'com.android.library'
 apply plugin: 'org.jetbrains.kotlin.android'
 
 android {
-  compileSdkVersion 30
+  compileSdk 31
   defaultConfig {
     minSdkVersion 21
   }
 
   compileOptions {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
```

Comment:
Not that it matters much for these samples, but shouldn't source and target compatibility for applications stay at Java 8?

Reply:
Not necessary for the sample!

Reply:
Oh man, I hadn't realized that Java 11 was already supported for source/target compatibility (through D8 I guess?).

Reply:
Yup!